Core/GDCore/Extensions/Builtin/MouseExtension.cpp

```diff
@@ -38,6 +38,36 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsMouseExtension(
       .AddCodeOnlyParameter("conditionInverted", "")
       .MarkAsSimple();
 
+  extension
+      .AddCondition(
+          "IsMouseWheelScrollingUp",
+          _("The mouse wheel is scrolling up"),
+          _("Check if the mouse wheel is scrolling up. Use MouseWheelDelta "
+            "expression if you want to know the amount that was scrolled."),
+          _("The mouse wheel is scrolling up"),
+          _("Mouse and touch"),
+          "res/actions/mouse24.png",
+          "res/actions/mouse.png")
+
+      .AddCodeOnlyParameter("currentScene", "")
+      .AddCodeOnlyParameter("conditionInverted", "")
```

Comment:
conditionInverted should not be needed right? I'm removing this - adding this will stop the inverted condition to work because GDevelop assume that you're taking care of this in your JS/C++ functions.